test/intern/mixins/Tree.js

```diff
@@ -409,6 +409,22 @@ define([
 					}, null, 'Modification of child should not throw error');
 				});
 			});
+
+			test.test('collapse items removed from store', function() {
+				return expand(0).then(function() {
+					assert.isTrue(grid.row(0).element.classList.contains('dgrid-row-expanded'),
```

Comment:
Not all browsers we test have support for `classList`.  Use `domClass.contains` or a RegExp instead.